### ISSUE TITLE
Cannot use embedded checkout with Nuxt 3

### ISSUE THREAD

**Issue by WojtekTheWebDev** [bigcommerce/checkout-sdk-js#1950]
**Describe the bug**
Hi, I faced a problem wit this package. I tried to use it together with nuxt 3, however the build breaks after importing `embedCheckout` from `@bigcommerce/checkout-sdk` in my file. Error is not descriptive, I'm adding it to screenshot section.

**To Reproduce**
1. Start with the fresh nuxt 3 app
2. `import { embedCheckout } from "@bigcommerce/checkout-sdk";`
3. Run `yarn build`

**Expected behavior**
App should build as expected. As it's the JS SDK, it should work with the JS ecosystem.

**Screenshots**
![image](https://user-images.githubusercontent.com/35867383/232339826-d55e7f84-c8d3-4071-81f9-e4fe9e7b251c.png)

**Additional context**
I'm using Node v16.14.0 and macOS.

I think it's not related with just nuxt, this problem might happened in the apps based on vite (but that's only an idea, I didn't verified that)

**Comment by daguitosama** [comment]
I just use it fine on Remix, maybe it’s something about Vite, but you could roll the CDN script in the mean time. 
Do you know about a good set of instructions on how to build a embedded checkout with this lib ? Im finding a little hard to follow the official docs. It does not explain much.

**Comment by sintj** [comment]
@WojtekTheWebDev have you find a solution to this? I'm building an e-commerce and having a hard time trying to implement a checkout with nuxt 3

**Comment by WojtekTheWebDev** [comment]
@sintj I didn't managed to solve this, however I was playing with it and I used just `iframe` to render this checkout and it works. A poc I was playing with [here](https://github.com/WojtekTheWebDev/bigcommerce-embedded-checkout-example). Unfortunately, without using this lib I don't think that you will be able to hook in `onSuccess` etc :( Based on my experience, I'd recommend using hosted bigcommerce checkout and redirect instead. Embedded one makes a lot of problems.

**Comment by daguitosama** [comment]
Yeah, to embed you need to match up domains, and even then, it could fail to load stuff

**Comment by WojtekTheWebDev** [comment]
@daguitosama In VueStorefront we have BigCommerce integration and we described how to set up BigCommerce to have a working embedded checkout in our [docs](https://docs.vuestorefront.io/bigcommerce/setup/configuring-bigcommerce.html). I missed your question about the set of instructions.

**Comment by bc-0dp** [comment]
Hi @WojtekTheWebDev was this resolved in the end?

Also this might be of interest to our Vue devs:
https://github.com/bigcommerce/checkout-sdk-js/pull/2252

**Comment by WojtekTheWebDev** [comment]
@bc-0dp after upgrading the dependencies, the project is building as expected, thanks!

My dependencies:

```json
"devDependencies": {
    "@types/node": "^20.11.19",
    "nuxt": "^3.10.2"
  },
  "dependencies": {
    "@bigcommerce/checkout-sdk": "^1.543.0",
  }
```
